components/Cards/src/MDCCardView.h

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
Per an offline discussion, MDCCardView will be removed in favor of:
MDCCard : UIControl
MDCCardCell : UICollectionCell